### PR TITLE
Add test when veileder is null in VeilederTilordnetServiceTest

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java [hunk 1]
```diff
@@ -45,6 +45,24 @@ void skal_oppdatere_tilordnet_veileder() {
         assertThat(bruker.isNy_for_veileder()).isTrue();
     }
 
+    @Test
+    void skal_oppdatere_tilordnet_veileder_med_null() {
+        final AktorId aktoerId = randomAktorId();
+        final VeilederId nyVeileder = VeilederId.of(null);
```
Comment (stianStensli) [inline src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java:51]: Are you sure that it wont be final VeilederId nyVeileder = null ?

Comment (dragutin-nav) [reply src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java:51]: VeilederId will be created, because of this line: https://github.com/navikt/veilarbportefolje/blob/dev/src/main/java/no/nav/pto/veilarbportefolje/domene/value/VeilederId.java#L9

Comment (stianStensli) [reply src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java:46]: ahh ok 🎉

Comment (dragutin-nav) [reply src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java:46]: I will write test for that one, to be sure

**Review by stianStensli — APPROVED** [review]

File: src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java (hunk 1 above)
Comment (dragutin-nav) [reply src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java:46]: hm, you are right..

Comment (dragutin-nav) [reply src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java:46]: @stianStensli I fixed that one, can you pls check again

File: src/main/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetDTO.java [hunk 2]
```diff
@@ -8,4 +10,15 @@
 public class VeilederTilordnetDTO {
     AktorId aktorId;
     VeilederId veilederId;
+
+    @JsonCreator
```
Comment (dragutin-nav) [inline src/main/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetDTO.java:14]: this constructor is here to support null values for veilederId

Comment (matiasvinjevoll) [reply src/main/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetDTO.java:14]: An alternative to this would be to use `@Data` instead of `@Value`, I think that would work, but then it's not immutable - tradeoff 🤔

**Review by matiasvinjevoll — APPROVED** [review]

File: src/main/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetDTO.java (hunk 2 above)
Comment (dragutin-nav) [reply src/main/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetDTO.java:14]: seems like json deserialization when `veilederId=null` is not working with `@Data`
